event-tool/src/main/java/uk/gov/justice/event/tool/StartTransformation.java

```diff
@@ -91,7 +91,7 @@ private void createTransformationTasks(final int pass) {
 
         final Stream<UUID> activeStreams = processAllStreams ? eventRepository.getAllStreamIds() : eventRepository.getAllActiveStreamIds();
 
-        logger.info(format("Processing %s streams", processAllStreams ? "all" : "active"));
```

Comment:
Using String.format is better as it allows you to test the actual message that would appear in the logs